### PR TITLE
Create tests for the individual file

### PR DESCRIPTION
**Pull request by Moohan**

### CODE REVIEW

File: R/process_tests_individual_file.R
Comment (lizihao-anu) [inline R/process_tests_individual_file.R:18]:
```suggestion
    write_tests_xlsx(sheet_name = "individual_file", year)
```

File: R/process_tests_individual_file.R
Comment (lizihao-anu) [inline R/process_tests_individual_file.R:11]:
```suggestion
  old_data <- get_existing_data_for_tests(data, file_version = "individual") %>%
      slfhelper::get_chi()

```

File: R/get_existing_data_for_tests.R
Comment (Jennit07) [inline R/get_existing_data_for_tests.R:53]: This function was not working quite right during testing. When calling an episode file it does not return recid and instead only has two columns `year` and Blank

File: R/process_tests_individual_file.R
Comment (Jennit07) [inline R/process_tests_individual_file.R:22]: See PR #682 and implement a similar change to reduce memory

**Comment by Jennit07** [comment]
Individual tests can be viewed at: "\\stats\hscdiip\SLF_Extracts\Tests\Jun_2023_2223_tests.xlsx"

The variables will be improved/match when the ongoing work by @lizihao-anu will merge but for now the tests are working. @Moohan I'll let you have another look before merging. Thanks

**Review by Jennit07 — APPROVED** [review]
See comments. Approving and will let @Moohan have another look at the code and the tests before merging

**Pull request merged**
